docs/docs/lib/node.mdx

```diff
@@ -547,14 +570,14 @@ console.log(result.experimentResult);
 
 ### Inline Experiments
```

Comment:
Do you think it'd be helpful to explicitly say these experiments aren't communicated back to GB? Or is it clear enough?

Reply:
Inline experiments fire a tracking callback just like ones that are used via feature flags.  The difference is that inline experiments are defined in code vs in the GrowthBook UI.  Inline experiments are only good for a few niche use cases.